### PR TITLE
Rosbridge

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
I have been having problems with the released version of rosbridge on multiple computers. So I think we should go back to the source version. Although if I am the only one, I should examine this.

### CODE REVIEW

File: python-autobahn/install.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- type: system
+  name: python-autobahn
```
Comment (ar13pit) [inline python-autobahn/install.yaml]: I don't think this is required by rosbridge.

**Comment by ar13pit** [comment]
I have been using the source version too. Didn't really try the system install.

File: python-autobahn/install.yaml (hunk 1 above)
Comment (MatthijsBurgh) [reply python-autobahn/install.yaml]: dep of rosbridge_server, https://github.com/RobotWebTools/rosbridge_suite/blob/ad63eb1f7a05d8d52470ac1364b033c74683bbbf/rosbridge_server/package.xml#L18

**Comment by ar13pit** [comment]
@MatthijsBurgh you are free to merge this.

**Pull request merged**